src/Core/Tool/Authorizer/UserAuthorizer.php

```diff
@@ -43,6 +43,12 @@ protected function getAllPrivs()
 	/* Authorizer */
 	public function isAllowed(Entity $entity, $privilege)
 	{
+		// Users cannot create 'owner' role
+		if (in_array($privilege, ['register', 'create', 'update'])
+			and $entity->role === 'owner') {
```

Comment:
Problem here is that an admin could edit the owner user and change their role to 'admin' .. I'm not sure how we avoid it. I gotta think about this a little